sensor/common/sensor/central_receiver_impl.go

```diff
@@ -87,6 +82,15 @@ func (s *centralReceiverImpl) receive(stream central.SensorService_CommunicateCl
 	}
 }
 
+func dropMessages(componentsQueues map[string]<-chan *central.MsgToSensor) {
+	for name, ch := range componentsQueues {
+		for msg := range ch {
+			log.Warnf("Dropping %s not handled by %s", msg.String(), name)
```

Comment:
I don't have a feel how many of those messages will be produced, but we saw in the past that customer panic when they see that something is dropped from a queue (they think, that they are loosing some data). Could we maybe make this a debug statement? Or if there are only few messages expected here, maybe we could reword this to sth benign like "flushing the queue remainder before shutdown" (feel free to adapt the wording).